scipy/linalg.pxd

```diff
@@ -1 +1 @@
-from .linalg cimport cython_blas, cython_lapack
+from scipy.linalg cimport cython_blas, cython_lapack
```

Comment:
The stated preference in #17234 was to keep this relative, to avoid that this ever picks up some other installed scipy. I understand that this isn't possible (or at least bugfree) with 0.29.x, but wouldn't want to mix these changes.

Reply:
Let me just say that that's not critical - whatever works here to get SciPy to build. We can deal with the rest later.

Reply:
It's probably still better to have a single commit with the absolute import changes to allow rolling it back independently at some point. Although there will likely be a couple of conflicts by then, but I'd guess that it would still help.